libs/langchain/langchain/llms/promptguard.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from typing import Any, Dict, List, Optional
+
+from pydantic import Extra, root_validator
+
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+from langchain.prompts.base import StringPromptValue
+from langchain.utils import get_from_dict_or_env
+
+logger = logging.getLogger(__name__)
+
+
+class PromptGuardLLMWrapper(LLM):
+    """
+    A LLM wrapper that uses the PromptGuard to sanitize the prompt before
+    passing it to the LLM, and desanitize the response after
+    getting it from the LLM.
```

Comment:
```suggestion
    An LLM wrapper that uses PromptGuard to sanitize the prompt before
    passing it to the LLM, and that desanitizes the response after
    getting it from the LLM.
```

Reply:
I am honestly unsure on the whole A vs An here, because if you read out LLM as "Large language model" then it should be A, but if you read it as "el el em" then I think "An" is technically accurate. Honestly though the fact there isn't actually a vowel visible makes my brain initially think An is wrong so I think I lean towards A (not that it really matters either way)

Reply:
should be an `an` according to my limited grammer knowledge but you are native speakers so you must be right, (and in this case have different opinions which makes me think that English grammer is sometimes too subtle).

Reply:
Did two searches. `A LLM` appears 7 times(excluding this one). `An LLM` appears 143 times.`An` wins the voting.